packages/client-sdk-nodejs/package.json

```diff
@@ -14,11 +14,11 @@
   },
   "scripts": {
     "prebuild": "eslint . --ext .ts",
-    "test": "jest --testPathIgnorePatterns auth-client.test.ts",
+    "test": "jest",
     "integration-test-auth": "jest auth-client.test.ts",
     "unit-test": "jest unit",
-    "integration-test": "jest integration --testPathIgnorePatterns auth-client.test.ts",
-    "build-and-run-tests": "cd ../common-integration-tests && npm run build && cd - && jest --testPathIgnorePatterns auth-client.test.ts",
+    "integration-test": "jest integration",
```

Comment:
I think this will also start running in the sdk canary: https://github.com/momentohq/canaries-kitchensink/blob/c9cdf00a6fe2eae18a3b8d180e0e05971de2d502/canaries/sdk-canary/index.ts#L117
Check if the session token will be available there